pkg/register/helper.go

```diff
@@ -14,5 +16,11 @@ import (
 // so it should probably be extracted to another common package
 func getPublicKeyFromBase58(publicBase58 string) (*ecdsa.PublicKey, error) {
 	publicKeyBytes := base58.DecodeAlphabet(publicBase58, base58.BTCAlphabet)
-	return ethcrypto.UnmarshalPubkey(publicKeyBytes)
+
+	curve := secp256k1.SECP256K1()
```

Comment:
I notice `proof/helper.go` has the same block. Is it worth factoring this unmarshal + make pub key bit at all?

Reply:
Thank you. I've refactored it to remove the two helper.go files. 0727f88